source/ada/lsp-ada_handlers.ads

```diff
@@ -117,6 +128,9 @@ private
 
       Refactoring : Refactoring_Options;
       --  Configuration options for refactoring
+
+      Open_Documents : Document_Maps.Map;
```

Comment:
Is it really needed? I don't see any references to this map in this patch

Reply:
yes, you need to expand the diff to `source/ada/lsp-ada_handlers.adb` - the github interface hides it by default because it's too large (big facepalm, github UI designers 🤦‍♀)

Reply:
Indeed  🤦‍♀